editor/plugins/script_editor_plugin.cpp

```diff
@@ -4380,6 +4380,8 @@ ScriptEditorPlugin::ScriptEditorPlugin() {
 	EDITOR_DEF("text_editor/external/exec_path", "");
 	EDITOR_DEF("text_editor/script_list/script_temperature_enabled", true);
 	EDITOR_DEF("text_editor/script_list/script_temperature_history_size", 15);
+	// A value of 1 doesn't make sense, as it would only highlight the currently selected script (and script temperature isn't visible on the current script).
+	EditorSettings::get_singleton()->add_property_hint(PropertyInfo(Variant::INT, "text_editor/script_list/script_temperature_history_size", PROPERTY_HINT_RANGE, "2,100,1"));
```

Comment:
A value of `100` is _ridiculously_ high though, is it not? Personal, of course.
```suggestion
	EditorSettings::get_singleton()->add_property_hint(PropertyInfo(Variant::INT, "text_editor/script_list/script_temperature_history_size", PROPERTY_HINT_RANGE, "2,50,1"));
```